doc/api/next_api_changes/behavior/22338-NH.rst

```diff
@@ -0,0 +1,5 @@
+rcParams['legend.loc'] now accepts float-tuple inputs
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The *legend.loc* parameter in *rcParams* now accepts float-tuple inputs, same as kwarg legend(loc...).
```

Comment:
This should be in `whats_new` not API change (this is a new feature that we did not used to have, not a change to existing behavior that might break a user).